src/domains/field/index.ts

```diff
@@ -25,7 +25,13 @@ export function Field(options?: FieldOptions): PropertyDecorator {
       ...options,
     };
 
+    const descriptor = Object.getOwnPropertyDescriptor(
+      targetInstance,
+      fieldName,
+    );
+
     fieldsRegistry.set(targetInstance.constructor, fieldName, {
```

Comment:
If it can be calculated from `targetInstance`, then passing it as another parameter is redundant in my opinion.